crates/fluvio-socket/src/versioned.rs

```diff
@@ -172,6 +172,17 @@ impl ClientConfig {
             use_spu_local_address: self.use_spu_local_address,
         }
     }
+
+    pub fn recreate(&self) -> Self {
```

Comment:
or Clone?

Reply:
you mean make DomainConnector clonnable or do it in a `impl Clone` for `ClientConfig`?

Reply:
Whichever is cleaner and simpler

Reply:
I need this PR first then https://github.com/infinyon/future-aio/pull/268

Reply:
Also https://github.com/infinyon/flv-tls-proxy/pull/34